src/snowflake/snowpark/mock/plan.py

```diff
@@ -438,6 +437,17 @@ def execute_mock_plan(
                         infer_type(cal_exp_res), nullable=True
                     )
             data.append(values)
+
+        if not children_dfs.indices:
+            aggregate_by_groups(child_rf)
```

Comment:
For my own education: what does `rf` mean?

Reply:
I'm guessing it's `result_from` or `result_frame`?

Reply:
It might be better to be more explicit, but anyways not within the scope of your change :)